react/components/ProductImages/global.css

```diff
@@ -5,133 +5,142 @@
   }
-}
 
-@media only screen and (min-width: 40em) and (max-width: 60em) {
-  .vtex-product-image__selected-image div {
-    width: 100%;
-    height: 300px;
+  .vtex-product-image__thumbnail-slider--vertical {
+    width: inherit;
   }
 
-  .vtex-product-image__zoom {
-    width: 100%;
-    height: 300px;
+  .vtex-product-image__thumbnail-slider--horizontal {
+    width: inherit;
   }
-}
 
-@media only screen and (max-width: 60em) {
-  .vtex-product-image {
-    width: 100%;
+  .vtex-product-image__thumbnail-slider {
+    width: inherit;
   }
-}
 
-@media only screen and (max-width: 40em) {
-  .vtex-product-image__thumbnail-slider-item,
-  .vtex-product-image__thumbnail-slider-item img {
-    max-height: 300px;
+  .vtex-product-image__thumbnail-slider .slick-slider {
+    width: inherit;
   }
-}
 
-@media only screen and (max-width: 27em) {
-  .vtex-product-image__thumbnail-slider-item,
-  .vtex-product-image__thumbnail-slider-item img {
-    max-height: 200px;
+  .vtex-product-image__thumbnail-slider-item {
+    height: 17em;
+    width: inherit;
   }
 }
 
 /* Arrows */
-.vtex-product-image .slick-prev .slick-disabled:before,
-.vtex-product-image .slick-next .slick-disabled:before {
-  color: inherit;
+.vtex-product-image__thumbnail-arrow {
+  display: inline-block;
+  width: 2.125em;
+  margin-right: 0.8125em;
 }
-
-.vtex-product-image .slick-prev:before,
-.vtex-product-image .slick-next:before {
-  font-size: 2rem;
-  color: inherit;
+.vtex-product-image__thumbnail-arrow:hover {
+  cursor: pointer;
 }
-
-.vtex-product-image
-  .vtex-product-image__vertical-thumbnail-slider
-  .vtex-slider__arrow-left,
-.vtex-product-image
-  .vtex-product-image__vertical-thumbnail-slider
-  .vtex-slider__arrow-right {
-  transform: rotate(90deg);
-  position: relative;
-  left: 6px;
-  right: 0px;
-  margin: auto;
-  margin-bottom: 15px;
+.vtex-product-image__thumbnail-arrow--vertical {
+  display: block;
+  margin: 0 auto;
 }
-
-.vtex-product-image .vtex-slider__arrow-left,
-.vtex-product-image .vtex-slider__arrow-right {
-  z-index: 1;
-  color: black;
+.vtex-product-image__thumbnail-arrow--vertical:not(.vtex-product-image__thumbnail-arrow--inverted) {
+  margin-bottom: 0.8125em;
 }
-
-.vtex-product-image
-  .vtex-product-image__horizontal-thumbnail-slider
-  .vtex-slider__arrow-left {
-  left: 0px;
+.vtex-product-image__thumbnail-arrow--vertical.vtex-product-image__thumbnail-arrow--inverted {
+  margin-top: 0.8125em;
 }
-
-.vtex-product-image
-  .vtex-product-image__horizontal-thumbnail-slider
-  .vtex-slider__arrow-right {
-  right: 10px;
+.vtex-product-image__thumbnail-arrow--inverted:not(.vtex-product-image__thumbnail-arrow--vertical) {
+  margin-right: 0;
+  margin-left: 0.8125em;
 }
 
 /* Dots */
 .vtex-product-image .vtex-slider__dots li button {
   color: inherit;
 }
-
 .vtex-product-image .vtex-slider__dots li button:before {
   font-size: 1rem;
   color: inherit;
 }
-
 :global .vtex-product-image .vtex-slider__dots li.slick-active button:before {
```

Comment:
I think that this  `:global`  is not necessary, try to remove it

Reply:
👍